### PR TITLE
add the color white

### PR DESCRIPTION
**Pull request by galaxine-senpai**
## Summary

This PR adds the color white to the colour.py file that for some reason did not exist before and had frustrated me as I looked through the docs so I added it myself
## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by GoogleGenius** [comment]
That's surprising that it was missing lol

File: discord/colour.py [hunk 1]
```diff
@@ -396,6 +396,14 @@ def yellow(cls) -> Self:
         .. versionadded:: 2.0
         """
         return cls(0xFEE75C)
+    
+       @classmethod
```
Comment (GoogleGenius) [inline discord/colour.py:400]:
```suggestion
   @classmethod
```

**Review by GoogleGenius — CHANGES_REQUESTED** [review]

File: discord/colour.py (hunk 1 above)
Comment (GoogleGenius) [reply discord/colour.py:400]: I think that was wrongly indented or something

Comment (galaxine-senpai) [reply discord/colour.py:400]: Ah, I didn't catch that thanks

**Comment by wasi-master** [comment]
There is probably a bug or something like that with the pure white color if I recall correctly. People used to suggest using a color close to pure white to mitigate the bug, I forgot the details though.

**Comment by Rapptz** [comment]
White used to be special cased by the client to be a theme specific colour that is either black or white depending on the theme the user did. On top of that, the colours used have always been historically colours that Discord uses within the client and their branding, that way the classmethods don't spiral out of control with a bunch of different colours. On top of that, black (0x000000) and white (0xFFFFFF) are pretty commonly known so they're not there.

**Comment by GoogleGenius** [comment]
Makes sense 👍

**Comment by galaxine-senpai** [comment]
Alright, thanks!

**Pull request closed without merging**